app/src/main/java/fr/free/nrw/commons/utils/ImageUtils.java

```diff
@@ -43,7 +43,11 @@
     public static final int IMAGE_WAIT = -2;
     public static final int EMPTY_TITLE = -3;
     public static final int FILE_NAME_EXISTS = -4;
+    public static final int FILE_FBMD = -5;
```

Comment:
I didn't notice this earlier but is the error code same for both FILE_FBMD and NO_CATEGORY_SELECTED? Please make FILE_FBMD = -6 then

Reply:
I am sorry, even I didn't noticed that :(